docs/rules/meta-satisfies-type.md

```diff
@@ -0,0 +1,55 @@
+# Meta should be followed by `satisfies Meta` (meta-satisfies-type)
+
+<!-- RULE-CATEGORIES:START -->
+
+**Included in these configurations**: (none)
+
+<!-- RULE-CATEGORIES:END -->
+
+## Rule Details
+
+This rule enforces writing `satisfies Meta` after the meta object definition. This is useful to ensure that stories use the correct properties in the metadata.
+
+Additionally, `satisfies` is preferred over type annotations (`const meta: Meta = {...}`) and type assertions (`const meta = {...} as Meta`). This is because other types like `StoryObj` will check to see which properties are defined in meta and use it for increased type safety. Using type annotations or assertions hides this information from the type-checker, so satisfies should be used instead.
+
+Examples of **incorrect** code for this rule:
+
+```js
```

Comment:
Rules should probably be separated for legibility, and use `ts` rather than `js` highlighting.